broadway/read-model-elasticsearch/0.3/config/packages/broadway_read_model_elasticsearch.yaml

```diff
@@ -0,0 +1,18 @@
+broadway:
+    read_model: broadway.read_model.elasticsearch.repository_factory
+
+services:
+    broadway.elasticsearch.client_factory:
+        class: Broadway\ReadModel\ElasticSearch\ElasticSearchClientFactory
+
+    broadway.read_model.elasticsearch.repository_factory:
+        class: Broadway\ReadModel\ElasticSearch\ElasticSearchRepositoryFactory
+        arguments:
+            - "@broadway.elasticsearch.client"
+            - "@broadway.serializer.readmodel"
+
+    broadway.elasticsearch.client:
+        class: Elasticsearch\Client
+        factory: ['@broadway.elasticsearch.client_factory', create]
+        arguments:
+            - "%elasticsearch%"
```

Comment:
This parameter is undefined. You should consider put it as an environment variable.

Reply:
Ping. Could you fix this and the PR is all good.

Reply:
Unfortunately this parameter is a nullable array so we cannot use environment variables here. I declared it and set it to null by default.

Reply:
@Nyholm are you ok with this?